pyperf/_utils.py

```diff
@@ -3,10 +3,11 @@
 import os
 import statistics
 import sys
+import sysconfig
 from shlex import quote as shell_quote   # noqa
 from shutil import which
 
-
+FREE_THREADING = bool(sysconfig.get_config_var('Py_GIL_DISABLED'))
 MS_WINDOWS = (sys.platform == 'win32')
 MAC_OS = (sys.platform == 'darwin')
```

Comment:
My suggestion is to add `USE_PSUTIL = FREE_THREADING` constant here and move comments explaining why Free Threading cannot use psutil right now, with the https://github.com/python/cpython/issues/116024 link.